addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -135,6 +159,25 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
         });
     },
 
+    /**
+     * Update the chatter.
```

Comment:
Could be the opportunity to state in docstring a bit why we do an update (not just what we do :) )